stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -428,6 +428,12 @@ module SwiftGlibc [system] {
         export *
       }
     }
+% if CMAKE_SDK in ["LINUX", "FREEBSD"]:
+    module sysexits {
+      header "${GLIBC_INCLUDE_PATH}/sysexits.h"
+      export *
+    }
+% end
```

Comment:
I have no idea why gyb is complaining here, but you should check that you haven't introduced any unnecessary invisibles or metachars.

Reply:
That was exactly it. Sorry about that!